frame/staking/src/lib.rs

```diff
@@ -297,6 +297,7 @@ pub mod migrations;
 pub mod slashing;
 pub mod weights;
 
+// TODO: set #[allow(hidden_glob_reexports)] once stable understands this term.
```

Comment:
Can we not just solve this instead of adding this?

Reply:
Good point - the explicit imports aren't too bad.